Makefile

```diff
@@ -14,16 +14,17 @@
 # limitations under the License.
 #
 
+# Make does not offer a recursive wildcard function, so here's one:
+rwildcard=$(wildcard $1$2) $(foreach d,$(wildcard $1*),$(call rwildcard,$d/,$2))
```

Comment:
@jstrachan - I double checked that his works for macOS. I am using 'GNU Make 3.81'. If it does not work for you, I think we can safely remove it. There is really no big benefit here. We can rely on Go figuring things out.